### PR TITLE
Dynamically render dog SVGs

### PR DESCRIPTION
**Pull request by AndreasWintherMoen**

### CODE REVIEW

File: frontend/components/svg/DynamicDogSvg.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import { DogBreedName } from '../../types/databaseTypes';
+import urlEncodeDogBreedName from '../../utils/urlEncodeDogBreedName';
+
+interface DynamicDogSvgProps {
+  dogBreed: DogBreedName;
+}
+
+export default function DynamicDogSvg({ dogBreed }: DynamicDogSvgProps) {
+  const name = urlEncodeDogBreedName(dogBreed);
+  const url = `https://api.winthermoen.no/images/bark/${name}.svg`;
```
Comment (haakonjacobsen) [inline frontend/components/svg/DynamicDogSvg.tsx:13]: Maybe we should add the base url to a config file somewhere.

File: frontend/utils/urlEncodeDogBreedName.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { DogBreedName } from '../types/databaseTypes';
+
+// convert from pascal case to kebab case and replace å with aa, ä with ae, ö with oe and æ with ae
+const urlEncodeDogBreedName = (breedName: DogBreedName): string =>
```
Comment (haakonjacobsen) [inline frontend/utils/urlEncodeDogBreedName.ts:4]: We should probably test all instances of possible dog names for this function.

Comment (AndreasWintherMoen) [reply frontend/utils/urlEncodeDogBreedName.ts:4]: True. Sounds like some really fun unit tests!

Comment (AndreasWintherMoen) [reply frontend/utils/urlEncodeDogBreedName.ts:4]: (to be implemented in the future...)

**Pull request merged**